From f702f12e42e4d14c73c531a41abc8398d6ffec28 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@apache.org>
Date: Mon, 19 Jun 2017 23:10:51 -0700
Subject: [PATCH 2525/2748] HADOOP-14515. Specifically configure
 zookeeper-related log levels in KMS log4j.

(cherry picked from commit 4cdae5d88ef7851ae284a115868dc4bbe24ae1bd)
(cherry picked from commit b89783e00e082e97c478862a2a257df1b3532a14)

Change-Id: I63a437ba07b240903e455a8076a9a5b379e2dd87
---
 .../hadoop-kms/src/test/resources/log4j.properties |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/src/test/resources/log4j.properties b/hadoop-common-project/hadoop-kms/src/test/resources/log4j.properties
index b347d27..e319af6 100644
--- a/hadoop-common-project/hadoop-kms/src/test/resources/log4j.properties
+++ b/hadoop-common-project/hadoop-kms/src/test/resources/log4j.properties
@@ -28,4 +28,7 @@ log4j.logger.org.apache.hadoop.crytpo.key.kms.server=ALL
 log4j.logger.com.sun.jersey.server.wadl.generators.WadlGeneratorJAXBGrammarGenerator=OFF
 log4j.logger.org.apache.hadoop.security=OFF
 log4j.logger.org.apache.directory.server.core=OFF
-log4j.logger.org.apache.hadoop.util.NativeCodeLoader=OFF
\ No newline at end of file
+log4j.logger.org.apache.hadoop.util.NativeCodeLoader=OFF
+# make zookeeper log level an explicit config, and not changing with rootLogger.
+log4j.logger.org.apache.zookeeper=INFO
+log4j.logger.org.apache.curator=INFO
\ No newline at end of file
-- 
1.7.9.5

